README.md

```diff
@@ -2,54 +2,80 @@
 
 This is a sample project for https://github.com/studds/nx-aws, powered by https://nx.dev
 
-## Example app
+### Example app
 
-This repo currently provides a single simple example app, with front and backends. The frontend uses angular. The backend using AWS SAM, and is built using nx-aws.
+This repo currently provides an example app highlighting how easy NX-AWS makes deploying frontend static sites and backend SAM applications from your mono-repo.
 
-## Development server
+The frontend (nx-aws-frontend) uses Angular (Could be React, nrwl NX has excellent support for React). 
 
-Frontend: `ng serve simple`
-Backend: `ng serve api`
+The SAM backend (nx-aws-api) depends on two lambda layers. 
```

Comment:
In this example repo, I wonder if it's worth including both a simple example (app + api) and a complex example (with layers)? Possibly an example where a binary dep is included directly?

Reply:
I think that's a good idea if we have the bandwidth, but I think in the first case the more complex example is better you can always delete the complexity.